res/skins/Shade/effectunit.xml

```diff
@@ -66,6 +69,23 @@
 						</Connection>
 					</WidgetGroup>	
 				</Children>		
+			</WidgetGroup>
+			<WidgetGroup>
+		        <ObjectName>EffectRightBorder</ObjectName>
+				<Size>0e,100f</Size>
+				<BackPath>style/style_bg_effectunit_right.png</BackPath>
```

Comment:
Bg image is not available.
git add -A ?
![shade-resize](https://user-images.githubusercontent.com/5934199/29781167-97f17cee-8c18-11e7-98fa-61766da79da6.png)